admin/app/views/workarea/admin/catalog_categories/edit.html.haml

```diff
@@ -60,6 +60,8 @@
             .property
               = label_tag 'category_default_sort', t('workarea.admin.fields.default_sort'), class: 'property__name'
               = select_tag 'category[default_sort]', options_for_select(@category.sort_options, selected: @category.default_sort)
+              %span.property__note
```

Comment:
You can check if there are any featured products here, so we only need to show them a note if that would actually affect this category.